src/tests/ConfigureRecordingSetTests.test.tsx

```diff
@@ -0,0 +1,176 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable jest/prefer-expect-assertions */
+
+import { cleanup, fireEvent, render, screen, waitFor } from '@testing-library/react';
+import i18next from 'i18next';
+import React from 'react';
+import { initReactI18next } from 'react-i18next/';
+
+import ConfigureRecordingSetPage from '../renderer/components/configure-recording-set-page/index';
+import { RecordingSet } from '../renderer/types';
+
+const flushPromises = () => new Promise(setImmediate);
+
+describe('configureRecordingSetPage', () => {
+  it('renders without showDetails button', async () => {
+    const originalError = console.error;
+    console.error = (...args: string[]) => {
+      if (/Warning.*not wrapped in act/.test(args[0])) {
+        return;
+      }
+      originalError.call(console, ...args);
+    };
+    i18next.use(initReactI18next);
+    render(
+      <ConfigureRecordingSetPage
+        onNext={(_event: React.MouseEvent<HTMLElement, MouseEvent>) => {}}
+        onBack={(_event: React.MouseEvent<HTMLElement, MouseEvent>) => {}}
+        onSetSelected={(_consumer: RecordingSet) => {}}
+      />,
+    );
+    let imageButtons = screen.getAllByRole('button');
+    expect(imageButtons).toHaveLength(4);
+    await flushPromises();
+    imageButtons = screen.getAllByRole('button');
+    expect(imageButtons).toHaveLength(4);
+    console.error = originalError;
```

Comment:
Shouldn't this be part of the `cleanup()`? Maybe create your own `cleanUp()` and wrap the library one inside of it?

Reply:
Also, there should be `beforeEach` and `afterEach` (or the equivalent) provided by jest, just for this purpose

Reply:
The linter is currently set to not allow beforeEach or afterEach hooks - I can create my own before and after functions and call them before and after each test though

Reply:
Interesting. What are the reasons it gives?